packages/core/src/components/portal/portal.tsx

```diff
@@ -99,7 +104,11 @@ export class Portal extends React.Component<IPortalProps, IPortalState> {
 
     public componentDidUpdate(prevProps: IPortalProps) {
         // update className prop on portal DOM element
-        if (this.portalElement != null && prevProps.className !== this.props.className) {
+        if (
+            this.portalElement != null &&
+            prevProps.className !== undefined &&
```

Comment:
this check might be overly narrow - if `prevProps.className === undefined`, we still might want to call `maybeAddClass()` in this block

Reply:
✅